pallets/claims/src/claims_data.rs

```diff
@@ -0,0 +1,5491 @@
+use sp_std::vec;
```

Comment:
remove this file, we dont have to lug it around anymore

Reply:
removed along with the migration part that was using the file